src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -1347,6 +1347,11 @@ protected override void ProcessRecord()
             {
                 foreach (string filename in expandedPaths)
                 {
+                    if (System.IO.Directory.Exists(filename))
```

Comment:
There is a using directive for `System.IO.Directory`, so any reason to use the full type name here?

Reply:
Fixed.